packages/react/src/components/TextInput/TextInput.tsx

```diff
@@ -220,7 +346,9 @@ const TextInput = React.forwardRef(function TextInput(
 });
 
 TextInput.displayName = 'TextInput';
+// @ts-expect-error types not yet provided for this component
 TextInput.PasswordInput = PasswordInput;
+// @ts-expect-error types not yet provided for this component
```

Comment:
I think this would still error out even if the components had been migrated, the error seems to be that those properties are not defined in `TextInput`
![image](https://user-images.githubusercontent.com/40550942/221666158-141f62d4-0933-44cd-b17d-d069103c63fd.png)